langfuse/api/resources/commons/types/trace_with_details.py

```diff
@@ -11,7 +11,10 @@
 
 class TraceWithDetails(Trace):
     observations: typing.List[str] = pydantic.Field(description=("List of observation ids\n"))
-    scores: typing.List[str] = pydantic.Field(description=("List of score ids\n"))
+    scores: typing.List[str] = pydantic.Field(
+            description=("List of score ids\n"), 
+            default_factory=list
```

Comment:
is this wanted?

Reply:
nope it is not

Reply:
removed it! the only reason was the pydantic parsing error because `score` was not being returned in the backend response.